python/Makefile

```diff
@@ -61,3 +61,7 @@ package: $(VENV_STAMP)
 .PHONY: edit
 edit:
 	$(EDITOR) $(ALL_PY_SRCS)
+
+.PHONY: clean
```

Comment:
should this not be called after the test has completed?

Reply:
@adityasaky and I were talking about this earlier, too. I think it's good practice to clean up after the tests, but for local development it could make sense to leave the virtual environment.

Reply:
ah ok sounds good.